src/main/kotlin/com/hibob/kotlinBasics/Main.kt

```diff
@@ -0,0 +1,41 @@
+package com.hibob.kotlinBasics//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
```

Comment:
General comment - please format your code. The indentations and spaces are not consistent. If you do `CMD + Option + L` in Intellij, it will format your code automatically, according to the IDE settings.